stories/locationAutoSuggest.story.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import LocationAutoSuggest from "../src/components/LocationAutoSuggest";
+import Input from "../src/components/Input";
+import { withState } from "@dump247/storybook-state";
+import { css } from "emotion";
+
+storiesOf("Components/LocationAutoSuggest", module).add(
+  "simple",
+  withState({ value: "" })(({ store }) => (
+    <>
+      <Input
+        onChange={value => store.set({ value })}
+        value={store.state.value}
+        placeholder="Enter Google Maps API Key"
```

Comment:
need to check if you need to mention what domain storybook is working on, so that the user knows if the API key will work or not.